src/Controller/CustomerController.php

```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Entity\Customer;
+use App\EntityRepository\CustomerRepository;
+use App\Model\Message;
+use App\Service\EmailSender;
+use App\Service\Messenger;
+use App\Service\SMSSender;
+use App\Service\Validator;
+use App\Service\WeatherService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ *
+ */
+class CustomerController extends AbstractController
+{
+
+    /**
+     *
+     * @Route("/customer/{code}/notifications", name="customer_notifications", methods={"GET"})
+     */
+    public function notifyCustomer(string $code, Request $request): Response
```

Comment:
If you wish to use CustomerRepository without autowiring, you can autowire managerRegistry as a 3rd parameter and use its getRepository() method to retrieve CustomerRepository. Otherwise, you can autowire CustomerRepository instead.